LICENSE

```diff
@@ -186,7 +186,12 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [Ulf Hamster, hamster@bbaw.de] 
+   Copyright 2020 Liao Ji-Lin, @windsound3482
```

Comment:
@windsound3482 please check your name. the contact details are optional (e.g. mail, github handle, etc.)